testdata/baselines/reference/submoduleAccepted/compiler/checkJsTypeDefNoUnusedLocalMarked.types.diff

```diff
@@ -10,9 +10,9 @@
 ->exports : (foo: Foo) => string
 ->(void 0) : FooFun
 +>module.exports = /** @type {FooFun} */(void 0) : (foo: typeof Foo) => string
-+>module.exports : any
-+>module : any
-+>exports : any
++>module.exports : (foo: typeof Foo) => string
```

Comment:
`typeof Foo` is correct -- the typeof-insertion insertion shenanigans in Strada were doing the wrong thing.